# Title

MIT 7.016 Introductory Biology, Fall 2018

Course 1

* human genome only 1.5% coding for proteins
* The DNA build blocks are the same
* A, T, C, G
* 1.8 meters DNA in one cell
* add color in genetic engineering
* observe different set of protein through color changes

---

Read: Text 2.2 and 3.1, recitation section

Textbook 2.2

* covalent bonds -> sharing electrons -> strongest -> e.g. $H_2$
  * The atoms can rotate -> make up living tissues
  * single bond, $H-H$
  * double bond, $C=C$
  * triple bond, $N≡N$
  * electronegativity, the attractive force that an atomic nucleus exerts on electrons
    * The closer the electrons, the greater
    * The more positive charges, the greater
    * Two atoms with close electronegativity -> nonpolar covalent bond
    * Two atoms with different electronegativity -> polar covalent bond
* ionic attractions -> achieve stability through gain or lose electrons
  * Significantly different electronegativity
  * Na + Cl -> $NA^+ + Cl^-$
  * Positively charged ions are cations
  * Negatively charged ions are anions
  * Groups of covalently bounded atoms with electric charge are complex ions; e.g. $NH_4^+$
  * $NaCl$ -> into water -> $Na^+$ attracts the $\delta^-$ pole of water and $Cl^+-$ attracts the $\delta^+$ -> water polarity, a good biological solvent
* hydrogen bonds -> weaker interaction
  * Resulted from $\delta^-$ and $\delta^+$
  * Hydrogen bonding between water -> important fro living system
  * Hydrogen bonds play important roles in determining and maintaining the three-dimensional shapes of giant molecules such as DNA and proteins
  * hydrophobic -> water-hating -> nonpolar molecules are more attracted to one another than to water
  * hydrophilic -> water-loving -> molecules with polar covalent bounds are attracted to polar water
* van der Waals forces
  * When two molecules are in close proximity

Text 3.1

* proteins
  * polymers, constructed by covalent bonding of monomers, smaller molecules
  * formed from different combinations of 20 amino acids, all of which share chemical similarities
* carbohydrates
  * polymers, constructed by covalent bonding of monomers, smaller molecules
  * can form giant molecules by linking together chemically similar sugar monomers to form polysaccharides
* nucleic acids
  * polymers, constructed by covalent bonding of monomers, smaller molecules
  * formed from four kinds of nucleotide monomers linked together in long chains
* lipids
  * noncovalent forces maintain the interactions between the lipid monomers
  * Also form large structures from a limited set of smaller molecules
* Polyer with molecular weights > 1000 -> macromolecules -> proteins, carbohydrates, nucleic acids
* Optical isomers
  * When carbon atom has four different atoms or groups of atoms attached to it
* The function of macromolecules are directly related to their three-dimensional shapes
  * Only nucleic acids specialize in information storage and transmission
  * most macromolecules are formed by condensation and broken down by hydrolysis

---

Course 2

* main brain bilayer
* H, C, N, O, P, S -> love to make covalent bonds -> cellular mass
* Phosphorus has 5 bond
* Functional group
  * hydroxy
    ```smiles
    OCC
    ```
  * carboxyl -> lose one $H^+$
    ```smiles
    CC(=O)O
    ```
  * amine -> get one $H^+$
    ```smiles
    NCC
    ```
  * phosphates
    ```smiles
    CCOP(=O)(O)O
    ```
  * thiol
    ```smiles
    CCS
    ```
  * amine + carboxyl -> amide (remove one $H_2O$)
    ```smiles
    CC(=O)N
    ```
  * hydroxy + carboxyl -> ester (remove one $H_2O$)
    ```smiles
    CC(=O)OCC
    ```
  * phosphates + hydroxy -> phosphate ester
    ```smiles
    CCOP(=O)(OCC)O
    ```
* Non-covalent bonding -> dynamics
  * To break a covalent, 90-100 Kcal/mol
  * To break non covalent, 1-10 Kcal/mol
  * Ionic bond (salt bridge)
  * [Hydrogen bond](https://www.khanacademy.org/science/hs-chemistry/x2613d8165d88df5e:states-of-matter/x2613d8165d88df5e:intermolecular-forces/v/hydrogen-bonding) -> pole -> protein
  * hydrophobic, exclusion of water
  * van der Waals, weakly polarized bonds

---

Read Text 3.2

* Amino acids are the building blocks of proteins (Amino group and Carboxyl group)
  ```smiles
  [NH3+]C[C-]([O-])=O
  ```
  * $-COOH -> -COO^- + H^+$
  * $-NH_2 + H ^+ -> -NH_3^+$
  * amino acids are both acids and bases
  * R groups determines the structure of it, so it determines the function of protein
  * There are 20 different R groups and the charged and polar ones are hydrophilic; the nonpolar ones are hydrophobic
  * disulfide bridge: two cysteine with oxidation
  * Peptide linkage: condensation by removing one $H_2O$ and connect Amino group and Carboxyl group
  * Hydrogen bonding determines the structure of polypeptide chains ($\delta^+$ of $N-H$ and $\delta^-$ of $C=O$)
* The tertiary structure of a protein is formed by bending and folding
* Heat protein -> Denatured -> break secondary and tertiary structure -> cool down -> in some cases, the protein returns to normal structure, meaning the shape is contained in its primary structure
* The quaternary structure of a protein consists of subunits, one tertiary structure
* The shape and surface further determines the protein function
  * Only certain shape of protein can fit into another shape of protein
* Molecular Chaperones protect proteins from inappropriate binding

---

Course 3

* Semipermeable membrane
  * wrapping up by lipids
  * $O_2$ and $H_2O$ can in and out freely
  * self-healing: seal from a hole
  * Electrostatic and hydrogen-bonding on the interface
* Protein formed by amino acids (40 ~ 2000)
* Always write from N to C (amino to carboxyl)
* motif: in secondary structure, the turns of the primary structure
* Proteins are condensation polymers of $\alpha$-amino acids
* Each protein is defined by a specific sequence of monomer amino acids linked by covalent bonding
* Most native proteins fold into stable secondary and tertiary structures
* Folding is defined by thermodynamic stability and the sum of many non-covalent interactions
* Protein subunit may also associate through non-covalent interactions into quaternary structures

---

Read text: 8.1, 8.3 - 8.5 (read the summary first, the summary is in the end of the chapter)

https://www.youtube.com/watch?v=yk14dOOvwMk

textbook 8.1

* The usable energy is free energy; The unusable energy is entropy
  * $H = G + TS$ (total energy = usable energy + unusable energy, T is absolute temperature and S is entropy)
  * $\triangle G_{reaction} = G_{product} - G_{reactants}$ => $\triangle G = \triangle H - T\triangle S$
* The potential energy is the energy of state; Kinetic energy is the energy of motion, which can do work
  * Potential: chemical bonds, concentration gradient, electric charge imbalance
  * Kinetic: heat
  * Potential <-> Kinetic
* Laws of thermodynamics
  * Energy cannot be created or destroyed
  * Energy transformations decrease the amount of energy for work and increase disorder
* The change in free energy determines its point of chemical equ. (The forward and reverse reactions proceed at the same rate)
* An exergonic reaction -> release free energy, -$\triangle G$; An endergonic reaction -> consumes free energy, +$\triangle G$
* Metabolism = the sum of all biochemical reactions; Catabolic reactions, breakdown of complex molecules and release energy; Anabolic reactions build complexity in the cell and are endergonic.
* Catabolic: Complex molecules -> free energy + small molecules
* Anabolic: Free energy + small molecules -> complex molecules
* Standard laboratory condition: $25^oC$, one atmosphere pressure, one molar concentration of the solutes, pH7. The standard free energy change under standard condition is $\triangle G^{o'}$

textbook 8.3

* The more negative the $\triangle G$ is, the further the reaction proceeds toward completion.
* Catalyst does not cause a reaction to occur, but merely increases the rate of the reaction
* To speed up a reaction, an energy barrier must be overcome
* Enzymes lower the energy barrier but do not affect equilibrium.

textbook 8.4

* Most enzymes are much larger than their substrates
* The active site is specific to the substrates
  * a substrate can be oriented correctly, chemically modified, or strained
* Induced fit: The change in enzyme shape caused by substrate binding
  * Induced fit explains why enzymes are so large
* Some enzymes require other cofactors to carry out catalysis
  * A coenzyme can be considered a substrate, it is changed by the reaction and then released from the enzyme
* Substrate concentration affects the rate of enzyme-catalyzed reaction

textbook 8.5

* Metabolism is organized into pathways in which the product of one reaction is a reactant for the next reaction.
* ENzymes can be regulated by inhibitors
  * If an inhibitor covalently binds to certain side chains at the active site of an enzyme, it will per- manently inactivate the enzyme by destroying its capacity to interact with its normal substrate.
  * The inhibition is reversible because if the con- centration of substrate is increased or if the concentration of inhibitor is reduced, the substrate is more likely to bind, and the enzyme is active again.
  * A competitive inhibitor binds to the active site, preventing substrate binding.
  * An uncompetitive inhibitor binds to the enzyme–substrate complex, preventing release of products.
  * A noncompetitive inhibitor binds at a site other than the active site, changing enzyme structure so that normal substrate binding cannot occur.
* The first reaction in a metabolic pathway is referred to as the commitment step.
* Enzymes are highly sensitive to changes in temperature and pH.
  * pH decreases -> more H+ -> react with $COO^-$ -> no longer negatively charged -> no longer interact with positively charged groups in the protein -> the folding of the protein may be altered
  * Temperature increase -> the reaction rate increase -> temperature too high -> reaction drops abruptly
  * Most enzymes in humans are more stable at high temperatures than those of the bacteria that infect us, so that a moderate fever tends to denature bacterial enzymes, but not our own.

---

Course 4

* Denaturation
  * Heat: boil eggs
  * pH: change the charges state
  * salts, organics: 
* $\alpha_2\beta_2$ protein
  * Normal DNA -> RNA (GAG) -> Normal Protein (GLU) -> Normal hemoglobin -> Normal Red Blood Cells
  * abnormal DNA -> RNA (GUG) -> MUTANT PROTEIN (VAL) -> Sickle cell -> Sicked red blood cells
  * Only 1/14 with sicked red blood suffer malaria

---

Learn carbohydrates

Textbook: 3.3

* Carbohydrates is not actually hydrates even though it has the form of $C_m(H_2O)_n$. The water molecules are not intact.
* Cells use glucose as energy source
  * Glucose exists in straight chains and in ring forms. Ring form is more stable.
* The absence of this oxygen atom is an important distinction between RNA and DNA. (five carbon sugar)
* maltose can be hydrolyzed into its monosaccharides in the hu- man body, whereas cellobiose cannot.
  * maltose = $\alpha-D-Glucose$ + $\beta-D-Glucose$
  * cellobiose = $\beta-D-Glucose$ + $\beta-D-Glucose$
* The different human blood groups (for example, the ABO blood types) get their specificities from oligosaccharide chains. 
* Both glycogen and starch are readily hydrolyzed into glucose monomers, which in turn can be bro- ken down to liberate their stored energy.
  * Glycogen store glucose in the liver and muscles
  * Large starch aggregates called starch grains can be observed in the storage tissues of plant seeds
* Cellulose, however, is chemically more stable because of its β-glycosidic linkages.

---

Course 5

* 25% massive macromolecules are carbohydrate
* HEXOSES (6C)
  * glycogen
* PENTOSE (5C)
  * 2-Deoxy Ribose -> DNA
  * Ribose -> RNA
* We have enzyme to break Glycogen but Cellulose. That's why we can get energy from Glycogen.

---

NUCLEIC ACIDS -> DNA/RNA
* DNA must exist as complementary strands
  * A <-> T (2 hydrogen bonds)
  * C <-> G (3 hydrogen bonds)

4.1 What are the chemical structures and functions of nucleic acids
* The unique functions of the nucleic acids—DNA and RNA—are information storage and transfer.
* Nucleic acids are polymers of nucleotides.
  * A nucleotide consists of three components: a nitrogen-containing base, a pentose sugar, and one to three phosphate groups 
* In DNA, the nucleotide bases are adenine (A), guanine (G), cytosine (C), and thymine (T). Uracil (U) replaces thymine in RNA.
* Complementary base pairing due to hydrogen bonds between A and T, A and U, and G and C occurs in nucleic acids.
* DNA is expressed as RNA in transcription. RNA can then specify the amino acid sequence of a protein in translation.
* RNA is usually a single strand.
* In contrast to RNA’s diversity in three-dimensional structure, DNA is remarkably uniform.
* DNA carries information and is expressed through RNA
  * The DNA sequences that encode specific proteins are transcribed into RNA.

13.1

* Griffith’s experiments in the 1920s demonstrated that some substance in cells can cause heritable changes in other cells.
  * Griffith concluded that in the presence of the dead S-type pneumococcus cells, some of the living R-type cells had been trans- formed into virulent S cells.
* Avery and his colleagues isolated the transforming principle from bacteria and identified it as DNA.
* The Hershey–Chase experiments established conclusively that DNA (and not protein) is the genetic material, by tracing the DNA of radioactively labeled viruses, with which they infected bacterial cells.
* Transformation and transfection can be studied with the aid of a genetic marker gene that confers a known and observable phenotype.
  * Any cell can be transfected, even an egg cell.

13.2

* Chargaff’s rule states that the amount of adenine in DNA is equal to the amount of thymine, and that the amount of guanine is equal to the amount of cytosine; thus the total abundance of purines (A + G) equals the total abundance of pyrimidines (T + C).
* X-ray crystallography showed that the DNA molecule is a double helix.
* Complementary base pairing between A and T and between G and C accounts for Chargaff’s rule. The bases are held together by hydrogen bonding.
  * DNA is usually a right-handed helix.
  * DNA is antiparallel
  * DNA has major and minor grooves in which the outer edges of the nitrogenous bases are exposed.
* Reactive groups are exposed in the paired bases, allowing for rec- ognition by other molecules such as proteins.

---

Course 6

* DNA, deoxyribose, ATGC, stable, dsDNA
* RNA, ribose, AUGC, various structure, rRNA, mRNA

---

Course 7

* Isotopes used as tracer and marker
  * You can locate the DNA with Isotopes.
  * When replication, new parts of DNA with non-isotopes were created. Through observing the position of isotopes and non-isotopes, we can understand the process of the replication.
  * The replication is a semi-conservative process
* Template driven polymer
  * AT place start the replication because there are three bonds here
  * helicase separates the strands of DNA
  * single strand binding protein attached to prevent single strand DNA bind again when replication.
  * Topoisomerase cut the DNA

---

Course 8

* transcription
  * bacteria -> metabolize phenylalanine from blood -> preclinical trials
* Bacteria replication error rate: 1/1000 (the speed 1000 BP/sec) -> unacceptable -> proof reading -> 3' exo nuclease -> error rate: $1/10^5$
* guardian of genome
  * The factors break the genome: hv, radiation, chemicals
  * base excision repair
  * nucleotide excision repair
* aging -> telomerase cannot fill the gaps of the replicated gene -> the gene getting shorter -> divide the cells to a certain amount, then it no longer to divide again
* The RNA sequence is synthesized in the 5' to 3' direction -> The DNA is read from 3' to 5' -> Promoter Sequences: Located upstream of genes, determining the start

---

Course 9

* RNA Only copy a part of the DNA sequence.
* (1) Promoters and enhancers pre mRNA -> mRNA, leave nucleus
* (2) Eukaryotic Transcription starts from Chromatin remodeling
  * DNA is wrapped around protein called histone (carries + charge from lysine, attracted by p group on the DNA)
  * enzyme stabilize the reaction through removing the + charge from histone with bonding with other molecules
* (3) 5' capping
* (4) polyadenylation of 3' -> Protect the mRNA
* (5) splicing
  * capped 5' - EXON - INTRONS - EXON - 3'AAAA
  * splicing the middle INTRONS -> 5' - EXON - EXON - 3'AAAA
  * Gives different functions of the proteins
* tRNA = decoders

---

Read 14.5

* During translation, amino acids are linked together in the order specified by the codons in the mRNA. This task is achieved by tRNAs, which bind to (are charged with) specific amino acids.
* Each tRNA species has an amino acid attachment site as well as an anticodon complementary to a specific mRNA codon. A specific activating enzyme charges each tRNA with its specific amino acid.
  * the mRNA codon for arginine is 5′-CGG-3′, and the comple- mentary tRNA anticodon is 3′-GCC-5′.
  * For example, the presence of inosine in the tRNA with the anticodon 3′-CGI-5′ allows it to recognize and bind to three of the alanine codons: GCA, GCC, and GCU. 
  * each mRNA codon binds to just one tRNA species, carrying a specific amino acid.
* The ribosome is the molecular workbench where translation takes place. It has one large and one small subunit, both made of ribosomal RNA and proteins.
  * A ribosome can use any mRNA and all species of charged tRNAs
  * When not active in the translation of mRNA, the ribosomes exist as two separate subunits.
* Three sites on the large subunit of the ribosome interact with tRNA anticodons. The A site is where the charged tRNA anticodon binds to the mRNA codon; the P site is where the tRNA adds its amino acid to the growing polypeptide chain; and the E site is where the tRNA is released.
* Translation occurs in three steps: initiation, elongation, and termination. The initiation complex consists of tRNA bearing the first amino acid, the small ribosomal subunit, and mRNA. A specific complementary sequence on the small subunit rRNA binds to the transcription initiation site on the mRNA.
  * the mRNA start codon in the genetic code is AUG 
* The growing polypeptide chain is elongated by the formation of peptide bonds between amino acids, catalyzed by the rRNA.
* When a stop codon reaches the A site, it terminates translation by binding a release factor.
  * The elongation cycle ends, and translation is terminated, when a stop codon—UAA, UAG, or UGA—enters. These codons do not cor- A respond with any amino acids, 
* In a polysome, more than one ribosome moves along a strand of mRNA at one time.

---

Course 10

* 4 bases (nucleotides) -> 20 amino acids
  * we need at least 3 bases to encode 20 amino acids (4^3 > 20)
* tRNA build hydrogen bond itself
  * Loop
  * Codon: triplet of nucleotides (where to mapping AA occur)
* genetic code
  * AUG -> start (Met)
  * stop: UAA, UAG, UGA
  * degenerate: > 1 codon/AA (some AA is more frequency, species specifics)
* load AA
  * AA attached on tRNA, costing one ATP

---

Course 11

* genetic DNA the same but different gene expression
* mRNA -> gene express -> different cell types
* endocytosis (outside -> in)
  * A stuff (cannot get into cell because of hydrophobic surface) -> lipid bilayer -> change the shape, make a hole in one the bilayer -> and take that stuff wrapped by lipid bilayer in
* exocytosis (inside -> out)
  * The reverse of the last point
* Mitochondria
  * outer membrane
  * inner membrane
  * retained DNA
  * divide by fission
  * three parents (two for cell, one for Mitochondria, repairing the disease from it)
* Cytoskeleton
  * dynamic machine in the cell -> with microtubules, generate force
  * microtubules -> biopolymers -> self assemble and grow and another end can shrink -> can pull -> pull chromosome both side -> chromosome split -> cell split

---

Read 5.3

* Eukaryotic cells are larger than prokaryotic cells and contain many membrane-enclosed organelles. The membranes that envelop organelles ensure compartmentalization of their functions.
* Organelles
  * microscopy
  * cell fractionation with chemical
  * many plant cell organelles are nearly identical in form to those observed in animal cells
* The endomembrane system is a group of interrelated organelles
  * is a series of interrelated compartments enclosed by membranes.
  *
* Some organelles transform energy
  * MITOCHONDRIA
  * chloroplasts
* A vacuole consists of a membrane-enclosed compartment full of water and dis- solved substances.
* The microfilaments, intermediate filaments, and microtubules of the cytoskeleton provide the cell with shape, strength, and movement.
* Motor proteins use cellular energy to change shape and move. They drive the bending movements of cilia and flagella, and transport organelles along microtubules within the cell.

---

Course 12

* haploid: single set of chromosomes
* dipolid: two complete sets of chromosomes (from each parents)
* homologous: same genes, different variants -> generate slight different protein -> different phenotypes
* Mitosis
  * All chromosomes lines up -> segregate equally
* Germ cells (Meiosis): produce gametes
  * two Meiosis: reductional division -> equational division
  * Don syndrome from wrong segregation

---

Course 13

* Mendel's theory
  * first law: segregation of chromosomes during Meiosis 1
  * second law: alignment of homologous chromosomes random
* sex linked inheritance
  * carried by a sex chromosome

---

Course 14

* Linkage: traits inherited together
* crossing over/ recombination: exchange of DNA between homologous chromosomes
* recombination frequency is proportional to the liner distance between genes
  * We need to consider the distance of crossover
* 2 genes unlinked, but linked to centromere -> 50% parental ditype, 50% non parental ditype
* 2 genes unlinked, unlined to centromere -> parental ditype: non parental ditype: tetratype = 1: 1: 4
* 2 genes linked, PD >> NPD+TT

---

Course 15

* Appearance/Behavior -> Gene
* 75% of human disease-causing genes are thought to have a homologue in fruit flies.
  * We use flies to find sleep disorder caused by gene

---

Course 16

* plasmid in bacteria
  * antibiotic resistance
  * Take a piece of eukaryotic and insert into the plasmid to use bacteria to clone the eukaryotic
* Steps cloning
  * cut DNA
    * restriction endonuclease
  * mix pieces
  * ligation
    * build covalent bond with enzyme
  * selection
    * DNA resistance strain
    * functional complementation: Functional complementation is a concept in genetics and molecular biology that refers to the phenomenon where the function of a defective or missing gene in an organism can be restored by providing a functional copy of that gene from another source.
      * auxotrophy- mutant that fails to make nutrient
      * prototrophy- produces all nutrients
* PCR
* Editing
  * CRISPER/cas9: RNA guided endonuclease

---

Course 17

* Linkage map
  * In order to see linkage, we need polymorphism or genetic variation
* molecular markers
  * simple sequence repeat
  * gel electrophoresis
    * The shorter DNA sequence go through faster
  * Microsatellite repeats used as molecular markers for mapping
* single nucleotide polymorphism
  * variation of single nucleotide at a position in genome
* restriction fragment length polym
* DNA sequencing (sanger method)
  * copy a piece with bacteria
  * extract them and add free DNA bases (ACGT), DNA polymerase, DNA primer, terminator bases
  * heat 93 degree -> two single strands
  * lower to 50 degree -> DNA primer to bind with strand
  * Increase to 60 degree -> enzyme (DNA polymerase) start from the primer and build the sequence again on one strand
  * The enzyme will be terminated by the terminator base (ddGTP, ddCTP, ddATP, ddTTP)
  * heat to 93 degree again -> The copied leaves from the strand
  * repeat the above process again and again and create lots of copies
  * put into the gel -> arrange from shortest to longest
  * Use laser to detect these arranged copied DNA strand 
    * The shortest one will be detect first and reveal the GCAT of it
    * Then the second shortest one
    * After all walk through the gel -> determines the sequence with colors

---

Course 18

* positional gene cloning: cloning a gene based on its chromosome position
  * linkage mapping -> chromosome position
  * physical map -> cloned pieces of DNA (encompassing chromosome region)
    * chromosome walk
  * identify interesting gene
    * is it transcribed in the tissue we are interested in
    * is there a conserved gene with interested function
* cDNA library
  * cDNA means complementary DNA, means complement of an mRNA transcript
  * use the reverse information like DNA -> RNA -> protein
  * RNA -> DNA
    * purify mRNA: use poly A tail -> use poly T tail to stick -> use A bead with poly T to drag mRNAs -> You got lots of mRNAs
    * With those RNAs, do reverse transcriptase -> get one strand DNA with RNA
    * degrade RNA, with RNAse H (enzyme) -> Have single strand DNA
    * Generate dsDNA -> not genomic DNA
      * no intrans
      * no promoter/enhancer, no non-transcribed region
      * poly A/T sequence
    * representation of genes in cDNA library is proportional to expression level of the gene
      * If the genotype is not express at all -> no protein -> not in cDNA library
* hybridization (microarray)
  * build matrix with different type of complementary RNA
  * Then get the genes in cancer and normal cells and mark color on it
  * Put those genes into those holes of the microarray
  * If more color of cancer it means this cancer may be caused by more of these gene
  * If more color of the normal cell, it means this cancer may be caused by lack of these gene

---

Course 19

* protein trafficking: how these protein know where to go
* where
  * inside cell
    * cytosol
    * organelles
      * mitochondria
      * nucleus
      * ...
  * outside
    * plasma membrane
    * secreted
* target the protein
  * targeting sequence
    * The sequence remains in the protein
  * post translational modification
    * lipidation: Lipidation is a natural post-translational modification where lipid molecules are covalently attached to proteins, influencing their localization, stability, and function within cells.
      * after lipidation attaches lipid molecules to a protein, it can help direct the protein to specific locations within the cell, such as cellular membranes or particular organelles, ensuring the protein is properly localized to perform its function.
    * pohspohorylation
    * ubiquitination
* NLS
  * get protein in and out the nucleus
* MLS
  * get protein in and out the mitochondria

---

Course 20

* chaperones proteins
  * help folding protect against misfolding
* proteasome
  * shredder, chops up proteins into peptide (8 - 14 amino acids)
* ubiquitin
  * tag a protein that should be destroyed by proteasome
* protein signaling
  * receive signal
  * transduce the signal
  * response
